.travis.yml

```diff
@@ -3,28 +3,28 @@ dist: trusty
 language: c
 matrix:
    include:
-     - addons: 
+     - addons:
           postgresql: 9.3
        env:
           - POSTGRESQL=9.3
        install:
            - sudo apt-get install postgresql-server-dev-9.3
            - make && sudo make install
```

Comment:
these lines can be moved to the `script` section below (see https://github.com/adjust/pg-language/commit/90bfe68f598d1ceda3ff3286206b272631d3b421 for an example)